src/plugins/bfetch/server/streaming/create_compressed_stream.ts

```diff
@@ -50,7 +50,6 @@ export const createCompressedStream = <Response>(
       }),
       finalize(() => {
         output.end();
-        sub.unsubscribe();
```

Comment:
This is where exception could happen - sub might be undefined on this point. This happens in case results observable is already completed. In the real world this could happen if bsearch is run with empty batch array. From Kibana client this could happen in case all requests were aborted before bsearch requests fired.